stdlib/email/headerregistry.pyi

```diff
@@ -90,12 +115,16 @@ class ContentTypeHeader(ParameterizedMIMEHeader):
     def value_parser(value: str) -> ContentType: ...
 
 class ContentDispositionHeader(ParameterizedMIMEHeader):
+    max_count: ClassVar[int | None]
```

Comment:
Not present at runtime, seems like it shouldn't be here either. It is inherited from `ParameterizedMIMEHeader`.

```suggestion
```